diff --git a/src/ipa/raspberrypi/meson.build b/src/ipa/raspberrypi/meson.build
index 32897e07..01416174 100644
--- a/src/ipa/raspberrypi/meson.build
+++ b/src/ipa/raspberrypi/meson.build
@@ -19,6 +19,7 @@ rpi_ipa_sources = files([
     'md_parser_smia.cpp',
     'cam_helper.cpp',
     'cam_helper_ov5647.cpp',
+    'cam_helper_mira220.cpp',
     'cam_helper_imx219.cpp',
     'cam_helper_imx290.cpp',
     'cam_helper_imx296.cpp',
